apps/crn-server/src/validation/user.validation.ts

```diff
@@ -2,7 +2,18 @@ import { userDegree, UserPatchRequest } from '@asap-hub/model';
 import { JSONSchemaType } from 'ajv';
 import { validateInput } from '.';
 
-const userPatchRequestValidationSchema: JSONSchemaType<UserPatchRequest> = {
+const userPatchRequestValidationSchema: JSONSchemaType<
```

Comment:
rather than define UserPathRequest as equal to UserPatchDataObject instead make sure that these properties are omitted on its definition as it never really receives them

Reply:
Done

Reply:
Done